<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-SQBL405" failures="1" tests="3" name="PetTest" time="13.379" errors="0" timestamp="2023-06-26T19:08:30 IST">
  <testcase classname="api.test.UserTests2" name="testPostUser" time="7.763"/>
  <testcase classname="api.test.UserTests2" name="testUpdateUserByName" time="2.695"/>
  <testcase classname="api.test.UserTests2" name="testDeleteUserByName" time="2.266">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests2.testDeleteUserByName(UserTests2.java:97)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- PetTest -->
